### PR TITLE
Update the CSS tokens used in mgt-teams-channel-picker

### PR DESCRIPTION
**Pull request by musale**

### CODE REVIEW

**Review by sebastienlevert — APPROVED** [review]

File: concepts/toolkit/components/teams-channel-picker.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ author: sebastienlevert
 
 # Microsoft Teams Channel Picker component in Microsoft Graph Toolkit
 
-A you can use the `mgt-teams-channel-picker` component to enable searches for Microsoft Teams channels associated with a user. The component can search all teams the user has joined, and each channel in those teams. 
+A you can use the `mgt-teams-channel-picker` component to enable searches for Microsoft Teams channels associated with a user. The component can search all teams the user has joined, and each channel in those teams.
```
Comment (Danielabom) [inline concepts/toolkit/components/teams-channel-picker.md:10]:
```suggestion
You can use the `mgt-teams-channel-picker` component to enable searches for Microsoft Teams channels associated with a user. The component can search all teams the user has joined, and each channel in those teams.
```

File: concepts/toolkit/components/teams-channel-picker.md [hunk 2]
```diff
@@ -34,63 +34,76 @@ Use the `selectChannelById(channelId: string)` method to programmatically select
...
 
 `mgt-teams-channel-picker` supports several [templates](../customize-components/templates.md) that you can use to replace certain parts of the component. To specify a template, include a `<template>` element inside a component and set the `data-type` value to one of the following.
 
-| Data type | Data context | Description |
-| --- | --- | --- |
-| loading | null: no data | The template used to render the state of the picker while the request to Microsoft Graph is being made. |
-| error | null: no data| The template used if user search returns no users. |
+| Data type | Data context  | Description                                                                                             |
+| --------- | ------------- | ------------------------------------------------------------------------------------------------------- |
+| loading   | null: no data | The template used to render the state of the picker while the request to Microsoft Graph is being made. |
```
Comment (Danielabom) [inline concepts/toolkit/components/teams-channel-picker.md:105]:
```suggestion
| loading   | null: no data | The template used to render the state of the picker while making a request to Microsoft Graph is being made. |
```

File: concepts/toolkit/components/teams-channel-picker.md [hunk 3]
```diff
@@ -34,63 +34,76 @@ Use the `selectChannelById(channelId: string)` method to programmatically select
...
+| error     | null: no data | The template used if user search returns no users.                                                      |
```
Comment (Danielabom) [inline concepts/toolkit/components/teams-channel-picker.md:106]:
```suggestion
| error     | null: no data | The template used if a user search returns no users.                                                      |
```

**Comment by musale** [comment]
> @musale, please review my comments. Thanks!

I have reviewed them and updated. Thank you for giving this a look.

**Comment by Danielabom** [comment]
Hi @musale! Thanks. Please review the build warnings. Thanks!

**Comment by musale** [comment]
> Hi @musale! Thanks. Please review the build warnings. Thanks!

The warnings seem to come from a project I don't work on. Isn't it out of scope for this PR?

**Review by Danielabom — APPROVED** [review]
PR Approved

**Pull request merged**